woocommerce/woocommerce
# Fix for 21214 - Regions text field not in line.

### All Submissions:






### Changes proposed in this Pull Request:



Closes #21214 

### How to test the changes in this Pull Request:

1. Go to shipping settings
2. Add a new shipping zone
3. See the text fields are aligned

### Other information:




### Changelog entry

> CSS Changes to make text fields in line on the shipping settings page
